assets/components/stripePopUpButton/stripePopUpButton.jsx

```diff
@@ -21,6 +21,7 @@ type PropTypes = {
   currency: Currency,
   email: string,
   isTestUser: boolean,
+  isPostDeploymentTestUser: boolean,
```

Comment:
This naming is used throughout, but I'm not sure that I like it... suggestions are welcome